CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+CHANGES IN VERSION 1.16.0 (FROM 1.15.1)
+===================================
+
+Appium 1.16.0 is a minor release
+
+#### General
+
+#### IOS (XCUITest):
+* Allow execution of WDA without app under test (https://github.com/appium/appium-xcuitest-driver/pull/1093)
+* Add `mobile: deleteFile` command for removing files/folders from the device (https://github.com/appium/appium-xcuitest-driver/pull/1095)
+* Fall back to `ios-deploy` if native install fails (https://github.com/appium/appium-xcuitest-driver/pull/1098)
+* Make sure ports are properly closed when cleaning up (https://github.com/appium/appium-xcuitest-driver/pull/1094)
+* Handle socket errors on session start (https://github.com/appium/appium-xcuitest-driver/pull/1101)
+* Properly handle creating simulators with patch versions (https://github.com/appium/node-simctl/pull/89)
+* Fix handling of Safari console log (https://github.com/appium/appium-remote-debugger/pull/176)
```

Comment:
I'd say one of the most significant changes in this release are the improvements in the safari remote debugger